### DISCUSSION TITLE
Old Book Positions (Alte Buch Optionen)

### DISCUSSION THREAD

**Discussion by mutoroglin** [PaulWoitaschek/Voice#1484] [Q&A]
I discovered this feature "Old Book Positions" (Alte Buch Positionen, I use the German version) and I love it.
But since I am a heavy user the list is endless and there is no possibility to search or copy.

I don't need a fancy features with this list added to voice, all I need is access to the text file to search it by myself.
Can you please tell where this information is stored?

Thank you!

**Comment by PaulWoitaschek** [comment]
What is your use case? This functionality was created to help with the migration to the new folder structure.
